stemcell_builder/stages/base_debootstrap/config.sh

```diff
@@ -20,6 +20,9 @@ then
 fi
 
 base_debootstrap_arch=amd64
+if [ `uname -m` == "ppc64le" ]; then
+  base_debootstrap_arch=ppc64el
+fi
```

Comment:
is there a global config.sh so we can set stemcell_arch=ppc64el and use it instead of checking uname everywhere?